testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/policy/PasswordPolicyTest.java

```diff
@@ -195,7 +199,8 @@ public void testRegexPatterns() {
 
             PasswordPolicy policy = null;
             try {
-                realmModel.setPasswordPolicy(PasswordPolicy.parse(session, "regexPattern"));
+                realmModel.setPasswordPolicy(PasswordPolicy.parse(session,
+                                                                  RegexPatternsPasswordPolicyProviderFactory.ID));
```

Comment:
Please remove this. Tests are by intent using a string and not a constant here.

Reply:
done